### PR TITLE
Fix: order book working with local currencies

### PR DESCRIPTION
**Pull request by lemunozm**
# Description

- Remove `DecimalConverter` trait, which does not handle Local currencies.
- Compute the min fulfillment amount directly in `order-book` generically for any currency.

### CODE REVIEW

File: pallets/order-book/src/lib.rs [hunk 1]
```diff
@@ -444,6 +437,8 @@ pub mod pallet {
 				Ok::<_, DispatchError>(*n)
 			})?;
 
+			dbg!(amount_out);
+			dbg!(min_fulfillment_amount_out);
```
Comment (wischli) [inline pallets/order-book/src/lib.rs:441]: Leftovers from debugging

**Review by wischli — COMMENTED** [review]
LGTM except for the two debugging leftovers. Thanks a lot for taking care of this so quickly! I propose to open a follow up PR which bumps spec versions.

**Review by wischli — APPROVED** [review]
Faster than Usain Bolt!

**Comment by lemunozm** [comment]
😆, thanks for reviewing William!

**Comment by lemunozm** [comment]
Not so fast, benchmarking is failing

**Comment by lemunozm** [comment]
Modified the fix a bit to use the native decimals instead of the native currency itself, to avoid difficult initializations of the native currency in benchmarking of pallets that uses order-book

**Review by wischli — APPROVED** [review]

**Pull request merged**